### PR TITLE
MNT: Re-rendered with conda-smithy 3.1.12 and pinning 2018.10.08

### PR DESCRIPTION
**Pull request by dpryan79**
Rerender and bump version

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -54,10 +54,10 @@ about:
     Analysis of dose-response data is made available through a suite of flexible and versatile
     model fitting and after-fitting functions.
   license_family: GPL2
-  license_file: '{{ environ["PREFIX"] }}\/lib\/R\/share\/licenses\/GPL-2'  # [unix]
-  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-2'  # [win]
```
Comment (bgruening) [inline recipe/meta.yaml]: Can you try just `LICENCE`.

Comment (dpryan79) [reply recipe/meta.yaml]: Sure

Comment (dpryan79) [reply recipe/meta.yaml]: I wonder if the cran skeleton should be tweaked to make that automatic.

Comment (bgruening) [reply recipe/meta.yaml]: To my knowledge, this is not standard and the pkg is "broken".

**Pull request merged**